pkg/extension/extensiontests/types.go

```diff
@@ -18,13 +18,13 @@ type ExtensionTestSpec struct {
 
 	// OtherNames contains a list of historical names for this test. If the test gets renamed in the future,
 	// this slice must report all the previous names for this test to preserve history.
-	OtherNames sets.Set[string] `json:"otherNames"`
```

Comment:
Enhancement currently relies on `originalName`, which cannot change. Every other name becomes, in effect, the current "display name". 
I'm not against storing the history, but I'm not confident there exists a performant "GROUP BY" mechanic to aggregate runs for all historical names using a list like this.

Reply:
Let's discuss on the enhancement https://github.com/openshift/enhancements/pull/1676/files#r1799952825